numpy/core/src/multiarray/datetime_strings.c

```diff
@@ -811,17 +656,21 @@ parse_iso_8601_datetime(char *str, Py_ssize_t len,
 
 parse_timezone:
     if (sublen == 0) {
-        if (convert_datetimestruct_local_to_utc(out, out) < 0) {
-            goto error;
-        }
-
-        /* Since neither "Z" nor a time-zone was specified, it's local */
+        /* by default, we no longer interpret times as local */
         if (out_local != NULL) {
-            *out_local = 1;
+            *out_local = 0;
```

Comment:
Might be better to rename the parameter from `out_local` to `out_naive`, keep it set to 1 here, and update the documentation comment for the function, etc.